wiki/common-misconceptions.md

```diff
@@ -0,0 +1,37 @@
+# Common Windows misconceptions
+
+This article covers many of the misinterpretations people may have with Windows and attempts to explain them. Seeing as this is a long article with a lot of required information that could not be condensed or cut out, it is recommended to link directly to individual sections.
+
+This does not cover all of the misconceptions - more sections will be added here as necessary.
+
+## RAM utilization
+
+Possibly the most common misunderstanding when it comes to Windows. Windows does not do memory management like any other OS. It preloads things that it thinks the user will use most into memory so it can be quickly accessed when other programs do not need the memory - this can lead to seemingly high idle memory utilization, and the user being alarmed. However, what the user doesn't know is that Windows will reallocate that memory holding data in standby to other programs or games if they so need it. Windows will not keep that memory allocated forever as that would lead to bad consequences such as system lock-ups, and no sane OS forgets to reallocate memory.
+
+For example: let's say we have `stuff.dll`, a massive 1 GB library of shared code. Windows knows that it commonly loads this file into memory and a lot of programs use it. If there's plenty of unused memory available, Windows will quietly load `stuff.dll` into memory and mark it as standby. If a program comes along and needs to use `stuff.dll`, instead of loading it from disk (which is a lot slower than the RAM bus), Windows directs it to the copy already in memory so it can skip loading it. It'll then be marked as in-use. After that program is done with it, it'll go back to being standby again. If a different program comes along and needs that space, Windows will freely allow it to overwrite `stuff.dll` as well as anything else in standby memory.
+
+Try loading up a memory intensive game, and taking a look at your total system memory utilization before and after launching the game. Let's say you are at 10 GB of total utilization before launching it, and the game is taking up about 6 GB. You'll see the total memory utilization only slightly creep up, possibly to 12 or 13 GB, not to 16 GB as you would expect. This is because Windows unloads stuff you don't need anymore to make room for the game's resources.
+
+RAM utilization is also dependent on how your RAM capacity - the more RAM you have, the more Windows uses to store frequently used code into standby memory. This is why some people with more memory notice higher utilization while some others with less memory notice significantly decreased utilization. Windows adapts based on how much memory you have and there is nothing to be concerned about if you notice high idle RAM utilization.
```

Comment:
The extra "how" here feels like a minor gramatical mistake/typo
```suggestion
RAM utilization is also dependent on your RAM capacity - the more RAM you have, the more Windows uses to store frequently used code into standby memory. This is why some people with more memory notice higher utilization while some others with less memory notice significantly decreased utilization. Windows adapts based on how much memory you have and there is nothing to be concerned about if you notice high idle RAM utilization.
```

Reply:
It is an error, thanks.

And oops I didn't realize I had to add those myself, doing it right now.

Reply:
Everything should be in order now.